data/authorization_plugins.yml

```diff
@@ -21,4 +21,13 @@
   author_name: ThoughtWorks Inc.
   contact_url: mailto:support@thoughtworks.com?subject=Info about LDAP/AD Authentication Plugin
   paid: true
+  
+- name: Atlassian Crowd Authentication Plugin
+  description: Delegates authentication services to Atlasian Crowd
+  readmore_url: https://gitlab.com/mgambini1975/go-crowd-plugin.git
+  author_url: http://www.ditechonline.it/
+  author_name: Di.Tech S.p.A.
+  contact_url: mailto:marco.gambini@ditechspa.it?subject=Info about Crowd Authentication Plugin
+  paid: false
```

Comment:
@mgambini , you can use any CI tool and any SCM platform as long you you can provide a jar that can be downloadable by users.